projects/js-packages/components/components/a8c-svg-title/index.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
```

Comment:
Would it make sense to use the WordPress abstraction from `@wordpress/element` instead of React itself here?

Reply:
Another component `JetpackLogo` depends directly on `react` as well, so not to make this PR more complicated. I'll discussion with @sergeymitr and see how we'll do it. Thanks